tests/e2e/test-data/formatters-data.ts

```diff
@@ -0,0 +1,61 @@
+export const formatters = [{
```

Comment:
Please add on top a short description of the file's purpose.

Reply:
thanks, updated